libcxx/include/map

```diff
@@ -642,28 +642,37 @@ class __map_value_compare {
   _LIBCPP_COMPRESSED_ELEMENT(_Compare, __comp_);
 
 public:
-  _LIBCPP_HIDE_FROM_ABI __map_value_compare() _NOEXCEPT_(is_nothrow_default_constructible<_Compare>::value)
+  _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX26 __map_value_compare()
```

Comment:
I guess you'll also need to update the synopsis with `//constexpr since C++26` wherever relevant.

Reply:
Fixed as of https://github.com/llvm/llvm-project/pull/134330/commits/c493c71e8505103b06af7dbd312b2f2a6230e1e4

Reply:
Added the signature as of : https://github.com/llvm/llvm-project/pull/134330/commits/f2a5a944e212127a9209042943c514e65b2e7890

let me know if that looks good!